test/hip-config-test/downstream/hip-config-test-user.cpp

```diff
@@ -0,0 +1,10 @@
+#include <iostream>
+
+#include "../tmp_install_dir/include/hip.hpp"
```

Comment:
> #include "../tmp_install_dir/include/hip.hpp"

This path looks weird to me since it has knowledge of details of the build system. Should the executable target have a `target_include_directories` so this does not end up in the file?